client/src/pages/MyPage.tsx

```diff
@@ -1,43 +1,90 @@
 import { useState } from 'react';
 import Layout from 'src/components/common/Layout';
+import ProductItemWrapper from 'src/components/common/ProductItemWrapper';
 import Chat from 'src/components/menu/Chat';
-import SalesList from 'src/components/menu/SalesList';
-import WishList from 'src/components/menu/WishList';
+import COLORS from 'src/constants/colors';
+import styled from 'styled-components';
 enum Menu {
   SalesList,
   Chat,
   WishList,
 }
```

Comment:
메뉴 리스트를 위한 새로운 배열을 만들지 않고 enum과 Object.values를 활용하는 건 어떨까요? 찾아보니 Object.values(enum)을 하면 결과가 배열로 나오더라구요.

```suggestion
enum Menu {
  SalesList = '판매목록',
  Chat = '채팅',
  WishList = '관심목록',
}

console.log(Object.values(Menu)) // ['판매목록', '채팅', '관심목록']
```

Reply:
수정하니까 코드가 굉장히 줄어드네요!! 좋은 리뷰 감사합니다